hardware/arduino/avr/cores/arduino/PluggableUSB.h

```diff
@@ -35,6 +35,7 @@ class PluggableUSBModule {
   virtual bool setup(USBSetup& setup) = 0;
   virtual int getInterface(uint8_t* interfaceCount) = 0;
   virtual int getDescriptor(USBSetup& setup) = 0;
+  virtual uint8_t getShortName(char *name) { name[0] = 'A'+pluggedInterface; return 1; }
```

Comment:
Yes we can surely improve, are you suggesting to do something like:

`name[0] = 'A' + ((uint8_t)pluggedInterface)`

?